src/collection-preferences/content-display/index.tsx

```diff
@@ -130,37 +130,20 @@ export default function ContentDisplayPreference({
         role="list"
       >
         <DndContainer
-          sortedOptions={sortedAndFilteredOptions}
-          getId={option => option.id}
-          onChange={options => onChange(options.map(({ id, visible }) => ({ id, visible })))}
-          disabled={columnFilteringText.trim().length > 0}
-          renderOption={props => (
-            <li
-              className={clsx(
-                getOptionClassName(),
-                props.isDragging && styles.placeholder,
-                props.isSorting && styles.sorting
-              )}
-              style={props.style}
-            >
-              <ContentDisplayOption
-                ref={props.ref}
-                listeners={props.listeners}
-                dragHandleAriaLabel={props.dragHandleAriaLabel}
-                onToggle={onToggle}
-                option={props.option}
-                disabled={props.attributes['aria-disabled']}
```

Comment:
From all attributes that DndContext provides the option only takes aria-disabled as we don't really need all like the role and tabIndex etc. However, it also does not take the aria-describedby which comes from the i18nStrings.dragHandleAriaDescription. As result, the ARIA description was not used at all.

Reply:
> the ARIA description was not used at all (an existing bug of collection prefs)

The ARIA description is passed to dnd-kit's `DndContext.screenReaderInstructions`, which enables screen reader announcements of the dragging instructions when focusing the drag handle. Is it necessary that it is an attribute of the drag handle as well?

Reply:
I checked in the html output for the ARIA description content and it looks like that it is only added as a hidden div which is not referenced from anywhere in the current main:

```
<div id="DndDescribedBy-2" style="display: none;">
  Use Space or Enter to activate drag for an item, then use the arrow keys to move
  the item's position. To complete the position move, use Space or Enter, or to
  discard the move, use Escape.
</div>
```

Reply:
That's how dnd-kit manages the announcement of these instructions, by just adding a visually hidden div and a live region to the doc ([link](https://github.com/clauderic/dnd-kit/blob/e9215e820798459ae036896fce7fd9a6fe855772/packages/core/src/components/Accessibility/Accessibility.tsx#L66-L69)). If we want to do use this string also as the ARIA description for the drag handle as well, won't we have them announced twice?

Reply:
As seen by the link that you shared, the description is not added as a live region - but as a hidden text. The hidden text is not announced to the screen-readers unless assigned with aria-labelledby or aria-describedby.

Reply:
Sure, I can dive deeper and do more changes but preferably not in scope of the current PR. I plan a few more upgrades to the dnd-container and can see if some improvements can be incorporated. So far the current behaviour looks ok - the description and the announcements are all there.